lib/px/service/client/hmac_signing.rb

```diff
@@ -0,0 +1,41 @@
+module Px::Service::Client
+  module HmacSigning
+    extend ActiveSupport::Concern
+
+    @@secret = ENV['PX_HMAC_SECRET_KEY']
```

Comment:
The rspec are failing because of this. This commit adds a new environment variable 'PX_HMAC_SECRET_KEY' and it is not set in Semaphore yet. Not sure if this is the best way to do it. 

Please inspect @cdmicacc